_includes/comments.html

```diff
@@ -0,0 +1,60 @@
+<div class="row">
+    <div class="center">
+        <div class="col-md-3">
+            <div class="center">
+                <h2 class="no-anchor">Comments </h2>
+            </div>
+            <div class="center share-buttons">
+                <ul>
+                    {% capture url %}{{site.url}}{{page.url}}{% endcapture %}
+                    <li>
+                        <a  href="https://www.facebook.com/sharer/sharer.php?u={{ url }}"
+                        onclick="window.open(this.href, 'pop-up', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"
+                        title="Share on Facebook" >
+                            <i class="sm-icon fa fa-facebook"></i>
+                        </a>
+                    </li>
+                    <li>
+                        <a  href="http://www.reddit.com/submit?url={{ url }}"
+                        onclick="window.open(this.href, 'pop-up', 'left=20,top=20,width=900,height=500,toolbar=1,resizable=0'); return false;"
+                        title="Share on Reddit" >
+                            <i class="sm-icon fa fa-reddit-alien"> </i>
+                        </a>
+                    </li>
+                    <li>
+                        <a  href="https://www.linkedin.com/shareArticle?mini=true&url={{ url }}&title={{ page.title }}&summary={{ page.description }}&source={{ site.title }}"
+                        onclick="window.open(this.href, 'pop-up', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"
+                        title="Share on LinkedIn" >
+                            <i class="sm-icon fa fa-linkedin"></i>
+                        </a>
+                    </li>
+                    <li>
+                        <a  href="https://twitter.com/intent/tweet?text={{ page.title }}&url={{ url }}"
+                        onclick="window.open(this.href, 'pop-up', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"
+                        title="Share on Twitter" >
+                            <i class="sm-icon fa fa-twitter"></i>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="http://news.ycombinator.com/submitlink?u={{ url }}&t={{ page.title | default:'' | uri_escape}}"
+                        onclick="window.open(this.href, 'pop-up', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"
+                        title="Share on Hackernews" >
+                            <i class="sm-icon fa fa-y-combinator"></i> 
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </div>
+    <script src="https://utteranc.es/client.js"
+        repo="gruntwork-io/guides-comments"
+        issue-term="pathname"
+        theme="github-light"
+        crossorigin="anonymous"
+        async>
+    </script>
```

Comment:
What a lovely, clean API 👍 

So was the only hold-up with utterances that pagination didn't seem to work?

Reply:
Yeah but now works. We needed to have at least 53 comments before pagination can work